nsxt/resource_nsxt_policy_project.go

```diff
@@ -0,0 +1,220 @@
+/* Copyright © 2020 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/protocol/client"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	infra "github.com/vmware/vsphere-automation-sdk-go/services/nsxt/orgs"
+)
+
+const defaultOrgID = "default"
+
+func resourceNsxtPolicyProject() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceNsxtPolicyProjectCreate,
+		Read:   resourceNsxtPolicyProjectRead,
+		Update: resourceNsxtPolicyProjectUpdate,
+		Delete: resourceNsxtPolicyProjectDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"nsx_id":       getNsxIDSchema(),
+			"path":         getPathSchema(),
+			"display_name": getDisplayNameSchema(),
+			"description":  getDescriptionSchema(),
+			"revision":     getRevisionSchema(),
+			"tag":          getTagsSchema(),
+			"short_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"site_info": {
+				Type: schema.TypeList,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"edge_cluster_paths": {
+							Type: schema.TypeList,
+							Elem: &schema.Schema{
```

Comment:
we can use getElemPolicyPathSchema here?

Reply:
Also for the T0 path?